### PR TITLE
🦴 README skeleton

### PR DESCRIPTION
**Pull request by marekkirejczyk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+- multiple contract calls to are combined into a single multi-call
+- extend with custom hooks
+- write integration tests for components and blockchain with ease
+
+## Example
+
+```ts
+function showBalance() {
+  const {account} = useEthers();
+  const balance = useTokenBalance(account.address);
```
Comment (Marik-D) [inline README.md:31]: Either change to `useEthBalance` or specify a token address

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+  return (<EthersProvider>
```
Comment (Marik-D) [inline README.md:32]: Doesn't make sense: `useEthers` hook should be used in a component rendered inside `EthersProvider`

**Review by Marik-D — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,53 @@
...
+# useDapp
+
+Framework for creating Dapp on Ethereum with React. 
+Easy. Secure. Testable. Extendable.
+
+## Challange 
+
+Writing Dapps comes with its own set of challenges and good practices:
+
+- refresh UI after each new bock arrival
```
Comment (JustynaBroniszewska) [inline README.md:10]:
```suggestion
- refresh UI after each new block arrival
```

File: README.md [hunk 4]
```diff
@@ -1 +1,53 @@
...
+- work in two modes: read-only (without wallet connected) and read/write (with wallet connected)
+- keep track of transactions status and inform a user when it is successfully mined or fails
+- work with different wallets (Metamask, Formatic, Authereum) and networks (mainnet and testnets) and handle change nicely
```
Comment (JustynaBroniszewska) [inline README.md:13]:
```suggestion
- work with different wallets (Metamask, Formatic, Authereum) and networks (mainnet and testnets) and handle these changes nicely
```

File: README.md (hunk 1 above)
Comment (marekkirejczyk) [reply README.md]: ✅

File: README.md (hunk 2 above)
Comment (marekkirejczyk) [reply README.md]: ✅

File: README.md [hunk 5]
```diff
@@ -1 +1,53 @@
...
+
+With all that in mind, we want our Dapps to be easy to test and extend, build with high-pace development and security in mind.
```
Comment (JustynaBroniszewska) [inline README.md:15]:
```suggestion
With all that in mind, we want our Dapps to be easy to test and extend, built with high-pace development and security in mind.
```

File: README.md [hunk 6]
```diff
@@ -1 +1,51 @@
...
+- work with different wallets (Metamask, Formatic, Authereum) and networks (mainnet and testnets) and handle these changes nicely
+
+With all that in mind, we want our Dapps to be easy to test and extend, built with high-pace development and security in mind.
+
+## Solution
+
+**useDapp** utilises best **React**, **Ethereum** and **testing** practices:
+- use react hooks to communicate with the blockchain
+- components automatically refresh on a new block
+- multiple contract calls to are combined into a single multi-call
```
Comment (JustynaBroniszewska) [inline README.md:22]:
```suggestion
- multiple contract calls are combined into a single multi-call
```

File: README.md [hunk 7]
```diff
@@ -1 +1,51 @@
...
+**useDapp** utilises best **React**, **Ethereum** and **testing** practices:
+- use react hooks to communicate with the blockchain
+- components automatically refresh on a new block
+- multiple contract calls are combined into a single multi-call
+- extend with custom hooks
+- write integration tests for components and blockchain with ease
+
+## Example
+
+```ts
```
Comment (JustynaBroniszewska) [inline README.md:28]:
```suggestion
```tsx
```

Comment (JustynaBroniszewska) [reply README.md:28]: This will colour code better :)

File: README.md [hunk 8]
```diff
@@ -1 +1,52 @@
-**Dapp framework**
+# useDapp
+
+Framework for creating Dapp on Ethereum with React.
+Easy. Secure. Testable. Extendable.
+
+## Challange
```
Comment (rzadp) [inline README.md:6]:
```suggestion
## Challenge
```

File: README.md [hunk 9]
```diff
@@ -1 +1,52 @@
...
+- extend with custom hooks
+- write integration tests for components and blockchain with ease
+
+## Example
+
+```tsx
+ReactDOM.render(
+  <EthersProvider>
+    <App />
+  <EthersProvider/>
```
Comment (rzadp) [inline README.md:33]:
```suggestion
  <EthersProvider/>,
```

**Review by rzadp — APPROVED** [review]

File: README.md [hunk 10]
```diff
@@ -1 +1,52 @@
...
+  document.getElementById('app')
+)
+
+function App() {
+  const balance = useEtherBalance();
+  return (
+    <div>
+    Your balance is: <span> {balance} </span>
+    </div>
+    )
```
Comment (Marik-D) [inline README.md:43]:
```suggestion
  )
```

**Review by Marik-D — APPROVED** [review]

**Pull request merged**